src/style.css

```diff
@@ -0,0 +1,104 @@
+@font-face {
+  font-family: Anevir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: AnevirBold;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body * {
+  /*stylelint-disable-next-line font-family-no-missing-generic-family-keyword*/
+  font-family: Anevir;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+body {
+  margin-top: 0;
+}
+
+form[data-qa="big"],
+form[data-qa="small"] {
+  margin-top: 20px;
+}
+
+.big,
```

Comment:
what is .big... is it form, or input, or something else?

Reply:
.big & .small are `input:text` that I've add at very first steps of doing this task

Reply:
let's rename it)